packages/run-protocol/src/proposals/econ-behaviors.js

```diff
@@ -267,7 +267,6 @@ export const setupReserve = async ({
   },
 }) => {
   trace('setupReserve');
-  console.trace('setupReserve');
```

Comment:
ah. that makes sense.

Perhaps the `console` provided by SwingSet doesn't have `.trace`.